### PR TITLE
Capstone project draft 1

### PR DESCRIPTION
**Pull request by RodChak**
##Basic Check✔

- Created a new branch according to the Github Flow rules.
- Added two HTML files that include the Home and About page.
- Added two .css files that include styles to the Home and About webpage.
- Set up linters for HTML/CSS and JS making sure they work correctly.
- Added a .gitignore file to make sure any irrelevant files are ignored by git.
- Added a professional README file description with the link of the webpage deployed using Github Pages.

##Project Requirements Basic checks ✔

- You should personalize the content of your page. Choose a topic that is different than the one in the original design. - **Done.**
- The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable. **Done.**
- Implement the About Page and The home page. - Done.
- The section "Featured speakers" should be created dynamically in JavaScript. - **Done**.

### CODE REVIEW

File: About.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="About.css">
```
Comment (danielufeli) [inline About.html]: - [ ] Nice working on your About us page, Kindly check the link to the style sheet, the page is not loading properly, the `stylesheet` is in the style folder, the link should be `style/About.css`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
```
Comment (danielufeli) [inline index.html]: - [ ] Nice working on your Index page, Kindly check the link to the style sheet, the page is not loading properly, the stylesheet is in the style folder, and the link should be `style/style.css`

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @RodChak , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights
- No Linter Errors✔️
- Clean Code ✔️
- Clean Design ✔️
- Descriptive Readme ✔️

## Suggested changes
[ ] - Kindly add your video presentation, a 5min recorded video presenting the features of your project.
[ ] - Kindly make use of GitHub Pages to deploy the web pages of your project and add a Live version link to your `Readme`

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by mwanawabangona — APPROVED** [review]
Hi @RodChak  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implimenting the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**